docs/start/framework/routing.md

```diff
@@ -301,6 +301,12 @@ You can destructure the `*`, you just have to assign it a new name. A common nam
 const { "*": splat } = params;
 ```
 
+## 404 Catch-All Route
+
+To create a route to catch non-matching routes, you can use the `route("*", "file-path")` route helper.
+
+If you use the `flatRoutes` convention, create a route file named `$.tsx`.
```

Comment:
```suggestion
```

Let's put this in [the file routing guide](https://github.com/remix-run/react-router/blob/main/docs/how-to/file-route-conventions.md) instead of here, which isn't about that topic.